static/js/components/subject_page/disaster_event_block.tsx

```diff
@@ -85,6 +84,7 @@ export function DisasterEventBlock(
     DEFAULT_FILTER_SECTION_HEIGHT
   );
   const [showFilters, setShowFilters] = useState(false);
+  const { fetchData } = useContext(DataFetchContext);
```

Comment:
I don't think we should/can disambiguate like that since it's the same function? 

DataFetchContext has a field fetchData which is set in the context component with the callback & called here in this line so that this component can use that function.

Reply:
ah ok. that makes sense to keep then